rest_framework/serializers.py

```diff
@@ -1081,7 +1081,10 @@ def save_object(self, obj, **kwargs):
                     self.save_object(related)
                 else:
                     # Reverse FK or reverse one-one
-                    setattr(obj, accessor_name, related)
+                    try:
+                        setattr(obj, accessor_name, related)
+                    except ValueError:
+                        getattr(obj, accessor_name).add(*related, bulk=False)
```

Comment:
https://docs.djangoproject.com/en/1.9/ref/models/relations/#django.db.models.fields.related.RelatedManager.add
> Changed in Django 1.9:
The bulk parameter was added. In older versions, foreign key updates were always done using save(). Use bulk=False if you require the old behavior.